Bio/GenBank/Scanner.py

```diff
@@ -719,8 +724,7 @@ def _feed_seq_length(self, consumer, text):
         consumer.size(length_parts[0])
 
     def _feed_header_lines(self, consumer, lines):
-        EMBL_INDENT = self.HEADER_WIDTH
-        EMBL_SPACER = " " * EMBL_INDENT
+        embl_indent = self.HEADER_WIDTH
```

Comment:
As noted elsewhere, all upper case was intended to indicate a constant: https://www.python.org/dev/peps/pep-0008/#constants